pyproject.toml

```diff
@@ -128,14 +123,60 @@ exclude = [
     'validation_operators/types/validation_operator_result\.py',                                           # 35
     'validation_operators/validation_operators\.py',                                                       # 16
     # tests
+    'tests/actions/test_core_actions\.py',
+    'tests/checkpoint/test_checkpoint_result_format\.py',
```

Comment:
These were incredibly tedious to add.
I should have just written a script to do it, but I decided I was already in too deep doing it manually. 😆